app/controllers/admin/communities_controller.rb

```diff
@@ -441,4 +468,21 @@ def ensure_white_label_plan
     end
   end
 
+  def update_feature_flags(user_enabled:, user_disabled:, community_enabled:, community_disabled:)
+    updates = []
+    updates << ->() {
+      FeatureFlagService::API::Api.features.enable(community_id: @current_community.id, person_id: @current_user.id, features: user_enabled)
```

Comment:
Suggestion: I'd pass those globals as parameter so that the function would me more pure and therefore easier to relocate somewhere else if needed.